pkgs/by-name/ex/exo/package.nix

```
@@ -0,0 +1,91 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  python3Packages,
+}:
+python3Packages.buildPythonApplication {
+  pname = "exo";
+  version = "0-unstable-2024-10-02";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "exo-explore";
+    repo = "exo";
+    rev = "2654f290c3179aa143960e336e8985a8b6f6b72b";
+    hash = "sha256-jaIeK3sn6Swi20DNnvDtSAIt3DXIN0OQDiozNUHqtjs=";
+  };
+
+  build-system = with python3Packages; [ setuptools ];
+
+  pythonRelaxDeps = [
```

This will break immediately, e.g. in https://github.com/NixOS/nixpkgs/pull/346048. Please use
```suggestion
  pythonRelaxDeps = true;
```

---

Thanks for the tip !
Addressed in #346514